test/e2e/volumes.go

```diff
@@ -169,6 +169,9 @@ func volumeTestCleanup(client *client.Client, config VolumeTestConfig) {
 		expectNoError(err, "Failed to delete client pod: %v", err)
 	}
 
+	By("sleeping a bit so client can stop and unmount")
```

Comment:
@rootfs lets see if maybe we can do it directly, using `waitForPodNoLongeRunningInNAmespace(f.Client, podName, ns)`, rather than a magic sleep ?